docs/features/vpn.rst

```diff
@@ -119,3 +120,50 @@ The resulting firmware will allow users to choose between secure (encrypted) and
 
 To confirm whether the correct cipher is being used, the log output
 of fastd can be checked using ``logread``.
+
+Wireguard
+^^^^^^^^^
+
+In order to support Wireguard in Gluon, a few technologies are glued together.
+
+**vxlan:** As Gluon typically relies on batman-adv, the Mesh-VPN has to provide
```

Comment:
```suggestion
**vxlan:** As Gluon typically relies on batman-adv, the mesh VPN has to provide
```

Reply:
We usually capitalize it as "Mesh VPN" in English.

Reply:
In names like DMVPN I could see why, but else?

Reply:
It would be a `mesh router`, a `mesh group`, why `Mesh VPN`?

Reply:
"mesh router" and "mesh group" sound more natural in my opinion, while "Mesh VPN" is a term we made up for a specific technology. Obviously subjective, but that's the reasoning.

Reply:
I went with Mesh VPN.